moveit_planners/pilz_industrial_motion_planner/include/pilz_industrial_motion_planner/pilz_industrial_motion_planner.h

```diff
@@ -138,6 +140,9 @@ class CommandPlanner : public planning_interface::PlannerManager
   pilz_industrial_motion_planner::JointLimitsContainer aggregated_limit_active_joints_;
 
   /// cartesian limit
+  std::shared_ptr<cartesian_limits::ParamListener> cartesian_limits_param_listener_;
+  cartesian_limits::Params cartesian_limits_params_;
```

Comment:
These can probably just be named `param_listener_` and `params_` unless you plan on there being other listener or parmas struct in the private members of this class.